### PR TITLE
homework4

### PR DESCRIPTION
**Pull request by artsiomshevel**
homework4

### CODE REVIEW

File: Homework_4/homework_4.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    console.log('Enter correct value')
+} else if (age < age_2) {
+    console.log("You don't have access cause your age is "  + age + " It's less then " + age_2)
+} else if ((age >= age_2) && (age < age_3)) {
+    console.log("Welcome  !")
+} else if (age > age_3) {
+    console.log("Keep calm and look Culture channel")
+} else {
+    console.log("Technical work")
+}
```
Comment (VladimirLQA) [inline Homework_4/homework_4.js]:
```suggestion
if (typeof(age) !== 'number') {
    console.log('Enter correct value')
} else {
  if (age < age_2) {
    console.log("You don't have access cause your age is "  + age + " It's less then " + age_2)
  } else if ((age >= age_2) && (age < age_3)) {
    console.log("Welcome  !")
  } else if (age > age_3) {
    console.log("Keep calm and look Culture channel")
  } else {
    console.log("Technical work")
}
```

File: Homework_4/homework_4.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+// 3.Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось, 
+//   преобразовываясь в number
+if (Number.isNaN(age) == false) {
```
Comment (VladimirLQA) [inline Homework_4/homework_4.js]:
```suggestion
if (isNaN(age)) {
```

isNaN return the boolean value, so not need compare with false or true

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN#difference_between_number.isnan_and_global_isnan

**Review by VladimirLQA — APPROVED** [review]

File: Homework_4/homework_4.js (hunk 1 above)
Comment (artsiomshevel) [reply Homework_4/homework_4.js]: fixed @VladimirLQA

File: Homework_4/homework_4.js (hunk 2 above)
Comment (artsiomshevel) [reply Homework_4/homework_4.js]: fixed @VladimirLQA

**Pull request merged**